_includes/flutter/files.md

```diff
@@ -0,0 +1,55 @@
+# Files
+
+There are three different file classes in this SDK:
+- `ParseFileBase` is and abstract class and is the foundation of every file class that can be handled by this SDK.
+- `ParseFile` (former the only file class in the SDK) extends ParseFileBase and is by default used as the file class on every platform but web.
+  This class uses a `File` from `dart:io` for storing the raw file.
+- `ParseWebFile` is the equivalent to ParseFile used at Flutter Web.
+  This class uses an `Uint8List` for storing the raw file.
+
+These classes are used by default to represent files, but you can also build your own class extending ParseFileBase and provide a custom `ParseFileConstructor` similar to the `SubClasses`.
+
+Have a look at the example application for a small (non web) example.
+
+When uploading or downloading a file, you can use the `progressCallback`-parameter to track the progress of the http request.
+```dart
+//A short example for showing an image from a ParseFileBase
```

Comment:
```suggestion
```

This code block annotation is not needed, the code block is already annotated by the surrounding documentation. To describe the code block, write the description as part of the docs.